### PR TITLE
Move config to PHP

### PR DESCRIPTION
**Pull request by wbloszyk**
Move config from XML to PHP by 1 to 1. Config update will be done in another PR.

### CODE REVIEW

File: src/DependencyInjection/SonataClassificationExtension.php [hunk 1]
```diff
@@ -38,15 +38,14 @@ public function load(array $configs, ContainerBuilder $container): void
         $config = $processor->processConfiguration($configuration, $configs);
         $bundles = $container->getParameter('kernel.bundles');
 
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('orm.xml');
-        $loader->load('form.xml');
-        // NEXT_MAJOR: Remove next line and the file.
```
Comment (VincentLanglet) [inline src/DependencyInjection/SonataClassificationExtension.php:44]: You lost this next major comment

Comment (wbloszyk) [reply src/DependencyInjection/SonataClassificationExtension.php:44]: This next major comment was for `serializer.xml`. I didnt remove it with drop ReST API.

Comment (wbloszyk) [reply src/DependencyInjection/SonataClassificationExtension.php:44]: https://github.com/sonata-project/SonataClassificationBundle/pull/735/files#diff-f7e94b9d2ece445736b81d6ab8d0b92cdec48382eb4a52d9fcd1e0e38e596025R46-R47

https://github.com/sonata-project/SonataClassificationBundle/pull/737/files#diff-f7e94b9d2ece445736b81d6ab8d0b92cdec48382eb4a52d9fcd1e0e38e596025L45-R44

**Review by VincentLanglet — APPROVED** [review]

File: src/Resources/config/form.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // Use "service" function for creating references to services when dropping support for Symfony 4.4
+    // Use "param" function for creating references to parameters when dropping support for Symfony 5.1
+    $containerConfigurator->services()
+
+        ->set('sonata.classification.form.type.category_selector', CategorySelectorType::class)
+            ->public()
+            ->tag('form.type', [
+                'alias' => 'sonata_category_selector',
```
Comment (ChristianGripp) [inline src/Resources/config/form.php:26]: Not related to this PR, but we should remove all form aliases

**Review by core23 — APPROVED** [review]

**Pull request merged**